spec/features/announcement_management_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'rails_helper'
```

Comment:
should be spec/course/announcement_management_spec.rb because I also have an announcement_management_spec.